docs/griptape-cloud/terms-of-service.md

```diff
@@ -1,74 +1,391 @@
+    accordance with the documentation included with the SDK and subject
+    to any restrictions and limitations specified in such documentation.
+
+6.  **RESTRICTIONS; USAGE**. Customer shall not: (a) modify,
+    disassemble, create derivative works of, translate, decompile,
+    reverse engineer, rent, lease, loan, transfer, or copy any portion
+    of the Griptape IPR; (b) input, upload, transmit, or otherwise
+    provide to or through the Griptape IPR any information or materials
+    (including Customer Data) that are unlawful or injurious, or
+    contain, transmit, or activate any virus, worm, Trojan horse,
+    malware, or other malicious computer code; (c) circumvent, disable,
+    or otherwise interfere with the Griptape IPR's operation, security,
+    or other technical features or measures of the Griptape IPR; (d) use
+    the Griptape IPR to develop a similar or competing product or
+    service; (e) perform or conduct any benchmarking or similar
+    activities (including publishing benchmarks or performance
+    information about the Griptape IPR); (f) transmit or upload any
+    viruses, Trojan Horses, backdoors, malware, or any other forms of
+    harmful or malicious materials to or through the Griptape IPR;
+    or (g) permit any third party to do any of the foregoing. Customer
+    shall use the Griptape IPR in compliance with all applicable laws
+    and regulations, any documentation regarding use of the Griptape IPR
+    as well as the terms of Griptape's Acceptable Use Policy (available
+    at https://docs.griptape.ai/griptape-cloud/acceptable-use) which is
+    incorporated herein by reference. Customer must not circumvent or
+    disable any security or other technological features of the Griptape
+    IPR. Customer agrees to comply with all access and/or usage
+    limitations that Griptape may make available from time to time.
+    Griptape may impose limitations on Customer's use of the Griptape
+    IPR, including limits on: (i) the number and rate of API requests
+    Customer may make to the Griptape IPR; (ii) the compute usage and
+    storage capacities used by Customer in connection with the Griptape
+    IPR, (iii) the number of Applications accessible to the
+    Customer; (iv) the quantity of data generated by Applications;
+    and (v) any other usage metrics that may be established by Griptape
+    from time to time.
+
+7.  **INTELLECTUAL PROPERTY.**
+
+    1.  **Ownership.** As between the parties, Griptape owns all right,
+        title, and interest, including all intellectual property rights,
+        in and to the Griptape IPR, including any improvements,
+        modifications, and enhancements to them. Except for those rights
+        expressly granted in this Agreement, no other rights to the
+        Griptape IPR are granted, either express or implied, to
+        Customer.** **
+
+    2.  **Customer Data. **Customer hereby grants to Griptape a
```

Comment:
Bold markdown closure \** is out of place, should be right after 'Data' before period?